lib/bundler/feature_flag.rb

```diff
@@ -64,6 +64,12 @@ def initialize(bundler_version)
       @bundler_version = Gem::Version.create(bundler_version)
     end
 
+    def github_https
```

Comment:
`github_https?`

Reply:
This could also probably be 

```ruby
settings_method(:github_https?, "github.https") { bundler_2_mode? }
```

That way you don't have to replicate the logic

Reply:
oh awesome, i didn't realize that. Thanks!